src/__tests__/components/EditorRightSideBar/FileManagement.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import FileManagement from "@/components/EditorRightSideBar";
+import renderWithMockedProvider from "../../utils";
+import { switchLoginModal, switchSideBar } from "../../../store/reducer/buttonToggleSlice";
+import { resetAll } from "@/store/reducer/canvasControlSlice";
+
+const mockDispatch = jest.fn();
+
+jest.mock("react-redux", () => ({
+  ...jest.requireActual("react-redux"),
+  useDispatch: () => mockDispatch,
+}));
+
+jest.mock("jspdf", () => {
+  return jest.fn().mockImplementation(() => {
+    return { save: jest.fn() };
+  });
+});
+
+describe("FileManagement", () => {
+  test("Each button in the File Management Component to display the correct text", () => {
+    renderWithMockedProvider(<FileManagement />);
+
+    const saveButtonElement = screen.getByTestId("save-btn");
+    const downloadButton = screen.getByTestId("download-btn");
+    const shareButtonElement = screen.getByText(/Share/i);
```

Comment:
Do we need to standardize "element" and "button"? I suggest using "button" consistently to maintain a cohesive UI effect.